etc/settings/settings_test.json

```diff
@@ -257,6 +257,16 @@
 		"default": "Verilog",
 		"arg": "netlist_lang"
 	    },
+	    "allow_unrelated_clustering": {
+                "label": "Allow unrelated clustering",
+                "widgetType": "checkbox",
```

Comment:
By default, both should be "checked" as in "on"

Reply:
This is done in upstream